console/v1/tests/consoleplugins.console.openshift.io/AAA_ungated.yaml

```
@@ -87,3 +87,137 @@ tests:
+    - name: Should throw an error for invalid CSP directive values, using wildcard
+      initial: |
+        apiVersion: console.openshift.io/v1
+        kind: ConsolePlugin
+        spec:
+          displayName: foo
+          backend:
+            type: Service
+          contentSecurityPolicy:
+          - directive: ScriptSrc
+            values:
+            - "*"
+      expectedError: "spec.contentSecurityPolicy[0].values[0]: Invalid value: \"string\": CSP directive value cannot be a wildcard"
+    - name: Should be able to create a ConsolePlugin with valid CSP value, using domain with wildcard prefix
+      initial: |
+        apiVersion: console.openshift.io/v1
+        kind: ConsolePlugin
+        spec:
+          displayName: foo
+          backend:
+            type: Service
+          contentSecurityPolicy:
+          - directive: ScriptSrc
+            values:
+            - "*.example.com"
+      expected: |
+        apiVersion: console.openshift.io/v1
+        kind: ConsolePlugin
+        spec:
+          displayName: foo
+          backend:
+            type: Service
+          contentSecurityPolicy:
+          - directive: ScriptSrc
+            values:
+            - "*.example.com"
+    - name: Should throw an error for invalid CSP directive values with whitespace, using space
+      initial: |
+        apiVersion: console.openshift.io/v1
+        kind: ConsolePlugin
+        spec:
+          displayName: foo
+          backend:
+            type: Service
+          contentSecurityPolicy:
+          - directive: ScriptSrc
+            values:
+            - https://scr ipt1.com/
+      expectedError: "spec.contentSecurityPolicy[0].values[0]: Invalid value: \"string\": CSP directive value cannot contain a whitespace"
+    - name: Should throw an error for invalid CSP directive values with whitespace, using tab
```

given following validation checks, it looks like we also need add more tests here
- name: Should throw an error for invalid CSP directive values with comma
- name: Should throw an error for invalid CSP directive values with semicolon
- name: Should throw an error for invalid CSP directive values with strings enclosed in single quotes(based on vojtechszocs's comment)